### PR TITLE
Update ltops kubes config to configure MySQL master/slaves, bump image to 4.10.1 and user count to 30k

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

**Review by crspeller — APPROVED** [review]

File: kubernetes/cluster_deploy.go [hunk 1]
```diff
@@ -54,23 +60,40 @@ type AppConfig struct {
...
 	Tag string `yaml:"tag"`
 }
 
 // TODO: Replace with an argument or config option when load test profiles are added
-const NUM_USERS = 10000
+const NUM_USERS = 30000
+
+const masterMySQLConfig = `
+[mysqld]
+log-bin
```
Comment (lieut-data) [inline kubernetes/cluster_deploy.go]: Should we converge on `log_bin` (and `super_read_only`) for configuration files, vs. using the allowed (but confusing!) command line flavours here?

File: kubernetes/cluster_loadtest.go [hunk 2]
```diff
@@ -57,6 +57,10 @@ func (c *Cluster) bulkLoad(loadtestPod string, appPod string) error {
 		return err
 	}
 
+	// If this command fails, assume user was already created
+	cmd = exec.Command("kubectl", "exec", appPod, "--", "./bin/platform", "user", "create", "--email", "success+ltadmin@simulator.amazonses.com", "--username", "ltadmin", "--password", "ltpassword", "--system_admin")
```
Comment (lieut-data) [inline kubernetes/cluster_loadtest.go:62]: If it does happen to fail, the reasons might be mysterious if we don't log something, akin to the `xmlsec1` issues recently resolved. Can we dump `cmd.CombinedOutput()` somewhere?

**Review by lieut-data — APPROVED** [review]

File: kubernetes/cluster_deploy.go (hunk 1 above)
Comment (jwilander) [reply kubernetes/cluster_deploy.go]: Yeah good point, I just copied what they were using in the myqlha chart but even if they're not being consistent we should be

File: kubernetes/cluster_loadtest.go (hunk 2 above)
Comment (jwilander) [reply kubernetes/cluster_loadtest.go:62]: Yep, I can log it out

**Pull request merged**